web/modules/custom/default_content_config/content/node/ad7aa7a5-33cf-400e-93cf-11820e467111.yml

```diff
@@ -39,7 +39,7 @@ default:
       pathauto: 0
   body:
     -
-      value: '<p><a href="/node/19" data-entity-type="node" data-entity-uuid="7086f5d5-7c6f-41ef-8a60-70cd267a9860" data-entity-substitution="canonical">RESOURCES</a>&nbsp;</p>'
+      value: '<p><a href="/node/14" data-entity-type="node" data-entity-uuid="fefddcf3-e162-4622-9176-a3f4c2dadd8e" data-entity-substitution="canonical">RESOURCES</a>&nbsp;</p>'
```

Comment:
This is strange. Why do we have a body field on topics? I don't see this in the spreadsheet nor in any of the source data https://docs.google.com/spreadsheets/d/100f8NjUlh86CzR5KhZ0Bph-Ihv6EyO2ilTrSz0SCDe4/edit?gid=641650981#gid=641650981
Can we just delete this field?

Reply:
Not sure what I'm missing, this is the config for a topic node - https://digitalgov.lndo.site/topics/accessibility

Reply:
Sorry, this doesn't really have anything do to with your PR. I was just pondering why there is even a body field on topics to start with.